pvr.hts/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.hts"
-  version="5.3.4"
+  version="5.3.5"
```

Comment:
Please increase the minor version number, not the micro as we are adding a new feature with this PR. => 5.4.0